examples/nlg_server/nlg_server.py

```diff
@@ -46,20 +56,13 @@ def generate_response(nlg_call, domain):
         template, tracker, channel_name, **kwargs)
 
 
-def create_app(domain):
-    app = Flask(__name__)
-
-    logging.basicConfig(level=logging.DEBUG)
-
-    @app.route("/nlg", methods=['POST', 'OPTIONS'])
-    def nlg():
-        """Check if the server is running and responds with the version."""
-        nlg_call = request.json
+@app.route("/nlg", methods=['POST', 'OPTIONS'])
+async def nlg(request):
+    """Endpoints which processes the Core request for a bot response."""
+    nlg_call = request.json
+    response = await generate_response(nlg_call, domain)
```

Comment:
either `generate_response` should be async, or there should be no `await`

Reply:
this was kind of implicit since the templategenerator in generate_response returned an awaitable. Made it explicit.